### DISCUSSION TITLE
Aerexplorer mod - retain Explorer ribbon AND remember last folder settings?

### DISCUSSION THREAD

**Discussion by explor202** [ramensoftware/windhawk-mods#1315] [General]
Does anyone know how to retain the W11 Explorer ribbon AND get W11 to remember the folder positions? When I restart my PC, I use an AutoHotKey script to open 23 different folders. 8 of those are opened by AHK on monitor 2, then it opens another on 2, moves it to 3 and closes it, then opens it again on 3. 8 more folders are then opened on 3 since Explorer remembers that's where the last folder was closed. The process is repeated for the other 6 folders on monitor 4, with the last folder, a "dummy" one, being opened, moved, and closed on 3 again, so all future folders are opened there. 

The "issue" I've run into is that I can only get this to work if I don't use the W11 ribbon. Explorer will always open each folder on the last known monitor, so I'd either have to edit each script to put in a move command, add a pause to each one so I have time to manually move each (necessary) folder, or just wait and move each one after the script has completely finished. 

My current workaround is to set Aerexplorer to disable the ribbon, run AHK at startup to get the folders open on their correct monitors, then switch Aerexplorer back to enable the ribbon. But surely there has to be a better solution somewhere, right?

Specs: HP EliteDesk 800 G6, Core i5-10500, 64GM RAM, 22H2 build 22621.4391

**Comment by m417z** [comment]
Tagging the mod author: @aubymori.

**Comment by explor202** [comment]
For anyone else looking for this same functionality, try WinSetView.
